### DISCUSSION TITLE
Is it possible to handle the Textbox width change event?

### DISCUSSION THREAD

**Discussion by sunsetninja** [fabricjs/fabric.js#7817] [Q&A] [answered]
## Version
5.2.1

The scaling event is not fired, there are no special events for Textbox in the documentation.

Is there an option to subscribe to this event?

**Comment by melchiar** [comment, marked as answer]
Changes to a textbox's width are handled in the `object:resizing` event, since it's not actually a scaling action.

**Reply by sunsetninja** [reply]
Got it, it works, thanks!
However, there is no such event in the documentation. :(

Also, is there any event that triggers when the resize is complete?
`object:resized` didn't work:)

**Reply by melchiar** [reply]
yes, that should be added to the docs.

As of Fabric v5, we've done away with the after object transform events in favor of the `object:modified` event. The event type is in the `action` property of the event (in this case it'll be action: "resizing")

**Reply by sunsetninja** [reply]
Okay, got it. Thanks for the clarification!
